### ISSUE TITLE
setVirtualContent fails with the following example

### ISSUE THREAD

**Issue by IagoLast** [WebReflection/hyperHTML#33]
```html
<!DOCTYPE html>
<html lang="en">

<head>
	<title>HyperHTML</title>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<script src="hyperhtml.js"></script>
</head>

<body>
</body>
<script>
var render = hyperHTML.bind(document.body);
var node1 = document.createElement('p');
var node2 = document.createElement('p');

node1.innerText = 'node1';
node2.innerText = 'node2';

render `<h1>Divs</h1>${[node1, node2]}<h1>end</h1>`;
//                               |----> This node wont be rendered
render `<h1>Divs</h1>${[node1, node1]}<h1>end</h1>`;
</script>

</html>
```

This results in the following. notice the missing `node1`

```html
<body>
  <h1>Divs</h1>
    <p>node1</p>
    <!--_hyper_html: 1579249636;-->
  <h1>end</h1>
</body></html>

```

**Comment by WebReflection** [comment]
correct, you cannot render the same node twice on the DOM.

Or better, if `[node1, node1]` is not a typo and meant to be like that, you probably need to understand how the DOM work. A node is unique, you want a clone? clone it then.

```js
render `<h1>Divs</h1>${[node1, node1.cloneNode(true)]}<h1>end</h1>`;
```

Otherwise you need two nodes. Indeed your first render just works

```js
render `<h1>Divs</h1>${[node1, node2]}<h1>end</h1>`;
```

TL;DR Having `[node1, node2]` is totally fine. Having `[node1, node1]` makes no sense at all.

`hyperHTML` works with real nodes, it doesn't use `innerHTML`

**Comment by WebReflection** [comment]
P.S. you have the same behavior with a document fragment, it's not about `hyperHTML`.

```js
var p = document.createElement(p);
p.textContent = 'node 1';
var f = document.createDocumentFragment();
f.appendChild(p);
f.appendChild(p); // you are just moving p at the end of the fragment

document.body.appendChild(p);
```

**Comment by IagoLast** [comment]
Wops, sorry I thought it was a bug. I'm just guessing how the whole project works and is amazing!

Thanks for the explanation
